README.md

```diff
@@ -41,14 +41,18 @@ $Env:Path += ";$(pwd)"
 
 ## Getting the Code and Building Electron
 
-After installing build-tools, you can run a new Electron build with this command:
+You can run a new Electron build with this command:
 
 ```sh
 # The 'Hello, World!' of build-tools: get and build `master`
 
 # On macOS if you aren't sure you have the right SDK version you should run
 e load-macos-sdks
 
+# Choose the directory where Electron's source and build files will reside.
+# You can specify any path you like. Two possibilities are
+# * `--root=~/electron-gn` if you're only going to use one branch
+# * `--root=~/electron-gn/branch` (e.g. `~/electron-gn/master`) if you're going to use multiple branches
 e init --root=/path/to/new/electron/directory --bootstrap testing
```

Comment:
Is there a reason we can't have this copy-pasteable as `e init --root=~/projects/electron ...` and let folks customize it if they want instead of forcing folks to basically type this out

Reply:
Not a bad idea

Reply:
done